### PR TITLE
Set brownouts for deprecated embed API v2

### PR DESCRIPTION
**Pull request by stsewd**
Ref https://github.com/readthedocs/readthedocs.org/issues/10677

### CODE REVIEW

File: readthedocs/embed/views.py [hunk 1]
```diff
@@ -66,8 +68,39 @@ def external(self):
         # Always return False because APIv2 does not support external domains
         return False
 
+    def _is_disabled_for_deprecation(self):
+        if settings.RTD_ENFORCE_BROWNOUTS_FOR_DEPRECATIONS:
+            return False
```
Comment (humitos) [inline readthedocs/embed/views.py:73]: This is for the tests, right? 🤔

File: readthedocs/embed/views.py [hunk 2]
```diff
@@ -66,8 +68,39 @@ def external(self):
...
+
+        tzinfo = pytz.timezone("America/Los_Angeles")
+        now = datetime.datetime.now(tz=tzinfo)
+        # Dates as per https://about.readthedocs.com/blog/2024/11/embed-api-v2-deprecated/.
+        # fmt: off
+        is_disabled = (
+            # 12 hours brownout.
+            datetime.datetime(2024, 12, 9, 0, 0, 0, tzinfo=tzinfo) < now < datetime.datetime(2024, 12, 9, 12, 0, 0, tzinfo=tzinfo)
+            # 24 hours brownout.
+            or datetime.datetime(2025, 1, 13, 0, 0, 0, tzinfo=tzinfo) < now < datetime.datetime(2025, 1, 13, 0, 0, 0, tzinfo=tzinfo)
```
Comment (humitos) [inline readthedocs/embed/views.py:83]:
```suggestion
            or datetime.datetime(2025, 1, 13, 0, 0, 0, tzinfo=tzinfo) < now < datetime.datetime(2025, 1, 14, 0, 0, 0, tzinfo=tzinfo)
```

**Review by humitos — APPROVED** [review]
Looks good. There is a typo on one of the dates, tho

File: readthedocs/embed/views.py (hunk 1 above)
Comment (stsewd) [reply readthedocs/embed/views.py:73]: yeah, but I should use `if not ...`

**Pull request merged**